### DISCUSSION TITLE
CSS Styling

### DISCUSSION THREAD

**Discussion by AlexKalopsia** [babybuddy/babybuddy#868] [Support]
Hi there, 

I'd like to know if there is a way to edit the styling of BB while running the docker container, without necessarily rebuilding the app.

**Comment by cdubz** [comment]
There is no specific mechanism for this. What are you looking to adjust? The design is pretty dated and could probably use some love. I'd be more inclined to look for contributors to help with that, but also open to coming up with some custom tweaks mechanism.

**Reply by AlexKalopsia** [reply]
Honestly i would just want to tweak overall styling: colors, text size, borders, etc. Even just the very basics could make a big difference.

Ideally this would all either be easily customizable via UI by having a "Custom CSS" field that takes priority over the default styling, or just by making sure there is an easy option to edit a custom CSS file in the repo.

BB is amazing, but i do wish it received some love when it comes to the looks, and I am happy to give a hand if I can

**Reply by cdubz** [reply]
I would _love_ someone with more visual experience/interest/skill/etc. then me to help out with the design 😁 So, if you're interested in that I'd rather work with you on PRs to improve the UI overall.

There are also a couple mobile focused U/X PRs in progress that could be related: babybuddy/babybuddy/pull/769, babybuddy/babybuddy/pull/754. Unfortunately, the contributor on those hasn't had the time to move them forward lately, but I was planning to work on picking up where they left off.

> Ideally this would all either be easily customizable via UI by having a "Custom CSS" field that takes priority over the default styling, or just by making sure there is an easy option to edit a custom CSS file in the repo.

That said, I'm open to this as well for folks who want to do further tweaking. I'd prefer the custom file approach over a field, but I assume it will be a bit more complicated.

**Reply by AlexKalopsia** [reply]
Yeah my thinking is that the custom field could be a "cheap" starting point and allow exploration and iteration before jumping into higher complexity redesign PRs
